### PR TITLE
Add Direct Debit test account details

### PR DESCRIPTION
**Pull request by m-green**
### Context
Service teams can test their Direct Debit integration in their test account by using a mock sort code and account number.

### Changes proposed in this pull request
Add a 'Test a mock bank account' section to our [Direct Debit documentation](https://docs.payments.service.gov.uk/direct_debit/#direct-debit).

### Guidance to review
Please check if factually correct.

### CODE REVIEW

File: source/direct_debit/index.html.md.erb [hunk 1]
```diff
@@ -241,6 +241,13 @@ You could then get a single page of these mandates by running:
 
 This would return payments 501 to 1000.
 
+## Test a mock bank account
+
+To test confirming bank account details as a user, use the following with your test account:
```
Comment (alexbishop1) [inline source/direct_debit/index.html.md.erb:246]: Not sure “confirming” is the right word here.

File: source/direct_debit/index.html.md.erb [hunk 2]
```diff
@@ -241,6 +241,13 @@ You could then get a single page of these mandates by running:
...
+
+- `200000` as the sort code
```
Comment (alexbishop1) [inline source/direct_debit/index.html.md.erb:248]: Maybe write the sort code as “20 00 00”. The style guide at https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bank-details says to put spaces in sort codes and we accept that format on the payment pages.

Comment (m-green) [reply source/direct_debit/index.html.md.erb:248]: Ah good point! I'll fix that, thanks Alex.

File: source/direct_debit/index.html.md.erb (hunk 1 above)
Comment (m-green) [reply source/direct_debit/index.html.md.erb:246]: Apologies, something went wrong here... this should be "entering". I'll fix now.

**Pull request merged**